src/components/CustomTooltip/CustomTooltip.jsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react"
+import { useState } from "react"
+import "./CustomTooltip.css"
+
+export default function CustomTooltip(props) {
+
+    const [isHovering, setIsHovering] = useState(false)
```

Comment:
I see that you are using react-bootstrap for some of the UI components. Does it not have a tool tip component? If it does, any particular reason to use a custom one?

Reply:
Yes, there is. Tooltip component (which would make it a lot easier), but one of the requirements was to make a custom cursor interaction so I created my own custom Tooltip